parametric/protos/apm_test_client.proto

```diff
@@ -10,6 +10,18 @@ service APMClient {
   rpc InjectHeaders(InjectHeadersArgs) returns (InjectHeadersReturn) {}
   rpc FlushSpans(FlushSpansArgs) returns (FlushSpansReturn) {}
   rpc FlushTraceStats(FlushTraceStatsArgs) returns (FlushTraceStatsReturn) {}
+
+  rpc OtelStartSpan(OtelStartSpanArgs) returns (OtelStartSpanReturn) {}
```

Comment:
The python api defines two methods for starting spans: start_span and start_as current span: https://opentelemetry-python.readthedocs.io/en/latest/api/trace.html#opentelemetry.trace.Tracer. If this distinction exists in other languages we should add an rpc for ‘startCurrentSpan’

Reply:
It does not in Java. Creating a span starts it, then you can call `makeCurrent()` on the instance.